examples/tty_conv.c

```diff
@@ -0,0 +1,161 @@
+/* PlanC (hubenchang0515@outlook.com) -- an example application
+ * that implements a custom conversation */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <termio.h>
+#include <security/pam_appl.h>
+
+/***************************************
+ * @brief echo off/on
+ * @param[in] fd file descriptor
+ * @param[in] off 1 - echo off，0 - echo on
+ * *************************************/
+static void echoOff(int fd, int off)
+{
+    struct termio tty;
+    (void) ioctl(fd, TCGETA, &tty);
+    if (off)
+    {
+        tty.c_lflag &= ~(ECHO | ECHOE | ECHOK | ECHONL);
+        (void) ioctl(fd, TCSETAF, &tty);
+    }
+    else
+    {
+        tty.c_lflag |= (ECHO | ECHOE | ECHOK | ECHONL);
+        (void) ioctl(fd, TCSETAW, &tty);
+    }
+}
+
+/***************************************
+ * @brief echo off stdin
+ * *************************************/
+static void echoOffStdin()
+{
+    echoOff(fileno(stdin), 1);
+}
+
+/***************************************
+ * @brief echo on stdin
+ * *************************************/
+static void echoOnStdin()
+{
+    echoOff(fileno(stdin), 0);
+}
+
+/***************************************
+ * @brief read a line input
+ * @return the input string
+ * *************************************/
+static char* readline()
+{
+    struct termio tty;
+    char input[PAM_MAX_RESP_SIZE];
+
+    flockfile(stdin);
+    int i = 0;
```

Comment:
Please avoid mixing declarations with statements.

Reply:
Is it means I shoud write as this?
```c
struct termio tty;
char input[PAM_MAX_RESP_SIZE];
int i;

flockfile(stdin);
for (i = 0; i < PAM_MAX_RESP_SIZE; i++)
```

Reply:
yes